<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectPlantDialog</class>
 <widget class="QDialog" name="ConnectPlantDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>29</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="hostnameLabel">
      <property name="text">
       <string>Hostname</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="hostnameLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="portLabel">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="portLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="clientIDLabel">
      <property name="text">
       <string>Client Identification</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="clientIDLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Connect to Plant Server</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
